source/docker/docker-installation.rst

```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out this section of the Wazuh documentation to learn about Docker installation: how to install the Docker engine and the Docker compose. 
```

Comment:
This section does not show how to install Docker compose. Instead it shows how to install the Wazuh solution using Docker compose for the deployment of the Wazuh manager and central components.